Gemfile.lock

```diff
@@ -0,0 +1,210 @@
+GEM
```

Comment:
I'm not sure that this is a good idea, maybe move this comment to a separate PR.